### DISCUSSION TITLE
Report minor typos

### DISCUSSION THREAD

**Discussion by KnightMiner** [SlimeKnights/slimeknights.github.io#1] [General]
Use this discussion to report minor issues with pages. Include the following info:

* **Page Link:**
* **Expected Text:**
* **Actual Text:**

**Comment by NyanMC** [comment]
![image](https://github.com/user-attachments/assets/a1efbea8-8330-4438-abba-df2510d1fa44)
Page Link: https://slimeknights.github.io/docs/json/stat-types/
Expected Text: Note vanilla makes no distinction between this tier and diamond, but mods may add blocks that require *netherite*.
Actual Text: Note vanilla makes no distinction between this tier and diamond, but mods may add blocks that require *gold*.

This typo also exists on the 1.18 version of the page.

**Comment by GamingRamon** [comment]
- Page Link: Modifiers and Traits (1.18)
https://docs.google.com/spreadsheets/d/17qwV8UOR0DBsUqbxyernG4e4LtjAZ55DMAqnJWc2kLw/edit?gid=0#gid=0
- Expected Text: Lacerating
- Actual Text: Sharp

**Comment by SilverMedal02** [comment]
Page Link: https://slimeknights.github.io/faq/best-tool/
Expected Text: “the best weapon” gives you nothing
Actual Text: “the best weapon” gives you _noting_

**Comment by NyanMC** [comment]
Page Link: https://slimeknights.github.io/faq/tinkers-construct-3/
Expected Text: This page covers frequently asked questions about Tinkers’ Construct *3*
Actual Text: This page covers frequently asked questions about Tinkers’ Construct *2*

**Comment by Xenon372** [comment]
Page Link: https://slimeknights.github.io/docs/roadmap/#projectiles
Expected Text: Bringing back projectiles with tool materials
Actual Text: Bringing bringing back projectiles with tool materials:

**Comment by ChiefArug** [comment]
Page: https://slimeknights.github.io/docs/json/loot-injectors/
Actual: Fluid texture JSON are defined under 
Expected: Loot Injectors are defined under
